Bulldozer.CSV/Maps/GroupImport.cs

```diff
@@ -0,0 +1,951 @@
+﻿// <copyright>
```

Comment:
Why is this named GroupImport.cs? Can we rename back to Group.cs? None of the other maps have the word Import on them. I know the diff will be bad, but we already reviewed GroupImport so it should be fine now.